mllm/models/qwen3/modeling_qwen3_qnn_aot.hpp

```
@@ -0,0 +1,355 @@
+    value_states = value_states.transpose(1, 2);
+
+    // [B, H, S, D]
+    query_states = q_rope_(query_states, llm_embedding_sin, llm_embedding_cos);
+    key_states = k_rope_(key_states, llm_embedding_sin, llm_embedding_cos);
+
+    // [B, H, S, D]
+    auto [key_states_new, value_states_new] = past_kv_cache->updateKVCache(layer_idx_, key_states, value_states);
+    key_states = key_states_new;
+    value_states = value_states_new;
+
+    Tensor attn;
```

Clang (14.0.6)

[error] variable 'attn' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)